babel.config.js

```diff
@@ -30,7 +30,7 @@ module.exports = function(api) {
         '@babel/preset-env',
         {
           forceAllTransforms: true,
-          useBuiltIns: 'entry',
+          useBuiltIns: 'usage',
```

Comment:
Note that this loads some polyfills automatically, which takes care of some of the heavy lifting for using things like new methods (`Object.assign`, etc).

It does so by automatically adding imports to [`core-js`](https://github.com/zloirock/core-js).

Notably, `core-js` does not include a `fetch` polyfill, which is why we have to add it manually.

See: https://github.com/zloirock/core-js#missing-polyfills

>`window.fetch` is not a cross-platform feature, in some environments, it makes no sense. For this reason, I don't think it should be in `core-js`. Looking at a large number of requests it _might_ be added in the future. Now you can use, for example, [this polyfill](https://github.com/github/fetch).